test/functional/test_helper.py

```diff
@@ -1,8 +1,8 @@
 def get_available_port_range(from_range, to_range):
     """Returns available local port number.
     """
-    if isinstance(from_range, int) or isinstance(to_range, int):
-        raise ValueError('from_range and to_range should be type of int')
+    if not isinstance(from_range, int) or not isinstance(to_range, int):
+        raise ValueError('"{}" and "{}" should be type of int')
```

Comment:
Just question, does it work?

```
ValueError('"{}" and "{}" should be type of int')
```

In my understanding, at least this works.

```
ValueError("{} and {} should be type of int".format(from_range, to_range))
```

Reply:
Yes, I've wrapped up in 917b34f